Xamarin.Forms.Platform.Tizen/Extensions/GeometryExtensions.cs

```diff
@@ -0,0 +1,191 @@
+
+            if (geometry is LineGeometry)
+            {
+                LineGeometry lineGeometry = geometry as LineGeometry;
+
+                path.MoveTo(
+                    Forms.ConvertToScaledPixel(lineGeometry.StartPoint.X),
+                    Forms.ConvertToScaledPixel(lineGeometry.StartPoint.Y));
+
+                path.LineTo(
+                    Forms.ConvertToScaledPixel(lineGeometry.EndPoint.X),
+                    Forms.ConvertToScaledPixel(lineGeometry.EndPoint.Y));
+            }
+            else if (geometry is RectangleGeometry)
+            {
+                FormsRectangle rect = (geometry as RectangleGeometry).Rect;
+
+                path.AddRect(new SKRect(
+                    Forms.ConvertToScaledPixel(rect.Left),
+                    Forms.ConvertToScaledPixel(rect.Top),
+                    Forms.ConvertToScaledPixel(rect.Right),
+                    Forms.ConvertToScaledPixel(rect.Bottom)),
+                    SKPathDirection.Clockwise);
+            }
+            else if (geometry is EllipseGeometry)
+            {
+                EllipseGeometry ellipseGeometry = geometry as EllipseGeometry;
+
+                path.AddOval(new SKRect(
+                    Forms.ConvertToScaledPixel(ellipseGeometry.Center.X - ellipseGeometry.RadiusX),
+                    Forms.ConvertToScaledPixel(ellipseGeometry.Center.Y - ellipseGeometry.RadiusY),
+                    Forms.ConvertToScaledPixel(ellipseGeometry.Center.X + ellipseGeometry.RadiusX),
+                    Forms.ConvertToScaledPixel(ellipseGeometry.Center.Y + ellipseGeometry.RadiusY)),
+                    SKPathDirection.Clockwise);
+            }
+            else if (geometry is GeometryGroup)
+            {
+                GeometryGroup geometryGroup = geometry as GeometryGroup;
+
+                path.FillType = geometryGroup.FillRule == FillRule.Nonzero ? SKPathFillType.Winding : SKPathFillType.EvenOdd;
+
+                foreach (Geometry child in geometryGroup.Children)
+                {
+                    SKPath childPath = child.ToSKPath();
+                    path.AddPath(childPath);
+                }
+            }
+            else if (geometry is PathGeometry)
+            {
+                PathGeometry pathGeometry = geometry as PathGeometry;
+
+                path.FillType = pathGeometry.FillRule == FillRule.Nonzero ? SKPathFillType.Winding : SKPathFillType.EvenOdd;
+
+                foreach (PathFigure pathFigure in pathGeometry.Figures)
+                {
+                    path.MoveTo(
+                        Forms.ConvertToScaledPixel(pathFigure.StartPoint.X),
+                        Forms.ConvertToScaledPixel(pathFigure.StartPoint.Y));
+
+                    Point lastPoint = pathFigure.StartPoint;
+
+                    foreach (PathSegment pathSegment in pathFigure.Segments)
+                    {
+                        // LineSegment
+                        if (pathSegment is LineSegment)
```

Comment:
I think we can extract the method using the polymorphism of LineSegment also.

Reply:
For the path segment, I think I will just leave it as now, because it is more readable in a point of maintenance and verification as we want to keep up with the other platform's GeometryExtensions`.